src/domains/citizen-auth-common/03_apim_v2.tf

```diff
@@ -234,3 +234,114 @@ module "api_fims_public" {
+  user_id             = "fastloginoperationuser"
+  api_management_name = data.azurerm_api_management.apim_v2_api.name
+  resource_group_name = data.azurerm_api_management.apim_v2_api.resource_group_name
+  first_name          = "PagoPA Operation"
+  last_name           = "PagoPA Operation"
+  email               = "area-assistenza-operations@pagopa.it"
+  state               = "active"
+}
+
+resource "azurerm_api_management_group_user" "pagopa_operation_group_v2" {
+  user_id             = azurerm_api_management_user.fast_login_operation_user_v2.user_id
+  api_management_name = data.azurerm_api_management.apim_v2_api.name
+  resource_group_name = data.azurerm_api_management.apim_v2_api.resource_group_name
+  group_name          = azurerm_api_management_group.api_fast_login_operation_v2.name
+}
+
+resource "azurerm_api_management_subscription" "pagopa_operation_v2" {
+  user_id             = azurerm_api_management_user.pagopa_user_v2.id
+  api_management_name = data.azurerm_api_management.apim_v2_api.name
+  resource_group_name = data.azurerm_api_management.apim_v2_api.resource_group_name
+  product_id          = module.apim_v2_product_fast_login_operation.id
+  display_name        = "Fast Login Operation API"
+  state               = "active"
+  allow_tracing       = false
+}
+
+
+
+# Named Value fn-fast-login
+resource "azurerm_api_management_named_value" "io_fn_weu_fast_login_operation_url_v2" {
+  name                = "io-fn-weu-fast-login-operation-url"
+  api_management_name = data.azurerm_api_management.apim_v2_api.name
+  resource_group_name = data.azurerm_api_management.apim_v2_api.resource_group_name
+  display_name        = "io-fn-weu-fast-login-operation-url"
+  value               = "https://io-p-weu-fast-login-fn.azurewebsites.net"
+}
+
+data "azurerm_key_vault_secret" "functions_fast_login_api_key" {
+  name         = "io-fn-weu-fast-login-KEY-APIM"
+  key_vault_id = module.key_vault.id
+}
+
+resource "azurerm_api_management_named_value" "io_fn_weu_fast_login_operation_key_v2" {
+  name                = "io-fn-weu-fast-login-operation-key"
+  api_management_name = data.azurerm_api_management.apim_v2_api.name
+  resource_group_name = data.azurerm_api_management.apim_v2_api.resource_group_name
+  display_name        = "io-fn-weu-fast-login-operation-key"
+  value               = data.azurerm_key_vault_secret.functions_fast_login_api_key.value
+  secret              = "true"
+}
+
+resource "azurerm_api_management_named_value" "api_fast_login_operation_group_name" {
+  name                = "api-fast-login-operation-group-name"
+  api_management_name = data.azurerm_api_management.apim_v2_api.name
+  resource_group_name = data.azurerm_api_management.apim_v2_api.resource_group_name
+  display_name        = "api-fast-login-operation-group-name"
+  value               = azurerm_api_management_group.api_fast_login_operation_v2.display_name
+  secret              = "true"
```

Comment:
Non necessary

Reply:
Can you add more detail?

Reply:
I don't think this is a secret value

Reply:
Fixed in [8f97212](https://github.com/pagopa/io-infra/pull/847/commits/8f972124028ad04b71e985e509d8d50073ea2154)